### DISCUSSION TITLE
No documentation about `fireImmediately`

### DISCUSSION THREAD

**Discussion by ftzi** [pmndrs/zustand#1320] [Q&A]
I couldn't find anywhere info about the `fireImmediately` property. Can someone explain to me what it does, and maybe add it to the Readme?

**Comment by dai-shi** [comment]
https://github.com/pmndrs/zustand/blob/5ca63d1d63b0ba104fbd3b6ce32c97be59fd9768/src/middleware/subscribeWithSelector.ts#L69-L71
If `fireImmediately` is truthy, it will invoke the callback once _in_ subscribe function.

---

We actually would like to remove `subscribeWithSelector` from readme.md and put more info in `docs/*`. cc @chrisk-7777

**Reply by ftzi** [reply]
So, this means the cb is called right after the subscribe is called?

**Reply by dai-shi** [reply]
Yes. More precisely, before subscribe reruns.
